charts/tekton-apps/templates/_snippets.tpl

```diff
@@ -201,3 +209,223 @@ the way you use it:
+{{- else }}
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxSurge: 0%
+    maxUnavailable: 100%
+{{- end }}
+
+replicaCount: 1
+
+{{- if hasKey .wordpress "smtp" }}
+smtpHost: {{ .wordpress.smtp.host }}
+smtpPort: {{ .wordpress.smtp.port }}
+smtpUser: {{ .wordpress.smtp.user }}
+smtpPassword: {{ .wordpress.smtp.password }}
+{{- else }}
+smtpHost: mailhog.mailhog.svc.cluster.local
+smtpPort: 1025
+smtpUser: {{ .project.project }}
+smtpPassword: anypassword
+{{- end }}
+
+{{ if hasKey .wordpress "nodeSelector" }}
+nodeSelector:
+  {{- range $k, $v := .wordpress.nodeSelector }}
+  {{ $k }}: {{ $v | quote}}
+  {{- end }}
+{{- else }}
+nodeSelector:
+  tech_stack: php
+  pvc: "true"
+{{- end }}
+
+{{ if hasKey .wordpress "podSecurityContext" }}
+podSecurityContext:
+  {{- range $k, $v := .wordpress.podSecurityContext }}
+  {{ $k }}: {{ $v }}
+  {{- end }}
+{{- else }}
+podSecurityContext:
+  enabled: true
+  fsGroup: 1000
+{{- end }}
+
+{{ if hasKey .wordpress "containerSecurityContext" }}
+containerSecurityContext:
+  {{- range $k, $v := .wordpress.containerSecurityContext }}
+  {{ $k }}: {{ $v }}
+  {{- end }}
+{{- else }}
+containerSecurityContext:
+  enabled: true
+  runAsUser: 1000
+{{- end }}
+
+{{ if ternary .wordpress.ci "true" (hasKey .wordpress "ci") }}
+initContainers:
+  - name: install
+    image: jnewland/git-and-stuff
+    imagePullPolicy: Always
+    command:
+      - bash
+      - -c
+      - |
+        git -c core.sshCommand="ssh -i ~/.ssh/id_rsa" clone {{ .wordpress.repository_ssh_url }} /tmp/wordpress
+        cp -Rf /tmp/wordpress/wp-content/* /bitnami/wordpress/wp-content/
+        cd /tmp/wordpress
+        git_branch=`git rev-parse --abbrev-ref HEAD`
+        git_log=`git log --format="%h - by %an - %ae - %s" HEAD~1..HEAD`
+        msg="[$git_branch] $git_log"
+        printf "%0.s-" $(seq 1 ${#msg})
+        echo
+        echo "$msg"
+        printf "%0.s-" $(seq 1 ${#msg})
+        echo
+        ls -la /tmp/wordpress/wp-content
+        echo "Folder wp-content copied from git repo into bitnami/wodpress"
+        echo "Done INIT"
+    volumeMounts:
+    - mountPath: /bitnami/wordpress
+      name: wordpress-data
+      subPath: wordpress
+    - mountPath: /home/app/.ssh/id_rsa
+      name: ssh-key
+      subPath: ssh-privatekey
+    - mountPath: /home/app/.ssh/known_hosts
+      name: github-known-hosts
+      subPath: config.ssh
+{{- if .wordpress.extraInitContainers }}
+{{ toYaml .wordpress.extraInitContainers | indent 2 }}
+{{- end }}
+
+extraVolumes:
+- name: ssh-key
+  secret:
+    secretName: {{ .wordpress.extraVolumesSshKeySecret | default (printf "%s-%s-deploy-key" .project.project .component.name) }}
+- name: github-known-hosts
+  configMap:
+    name: github-known-hosts
+{{- if .wordpress.extraVolumes }}
+{{- toYaml .wordpress.extraVolumes | nindent 0 }}
+{{- end }}
+{{- end }}
+
+livenessProbe:
+  httpGet:
+    path: /wp-admin/install.php
+    port: 8080
```

Comment:
Idk why, but sometimes I face  readness and readness probes via https:// proto to WP (it works on http), so continainer become unhealthy.   
```suggestion
    port: 8080
    scheme: HTTP
```